fbw-common/src/wasm/systems/systems/src/failures/mod.rs

```diff
@@ -13,7 +13,11 @@ pub enum FailureType {
     TrimAirOverheat(ZoneType),
     TrimAirFault(ZoneType),
     TrimAirHighPressure,
+    CpcFault(usize),
+    OutflowValveFault,
+    SafetyValveFault,
```

Comment:
Move these after `GalleyFans` to have them in the same order as everywhere else.